components/NavigationBar/src/MDCNavigationBar.m

```diff
@@ -261,6 +263,14 @@ - (void)encodeWithCoder:(NSCoder *)aCoder {
   [aCoder encodeInteger:self.titleAlignment forKey:MDCNavigationBarTitleAlignmentKey];
 }
 
+- (void)setTitleFont:(UIFont *)titleFont {
+  _titleFont = [titleFont fontWithSize:kTitleFontSize];
+  if (!_titleFont) {
+    _titleFont = [MDCTypography titleFont];
```

Comment:
We should add a follow up task here to capture the fact that mdc typography can be used to change the font size to a different size than 20. I see that you made the change here to be compatible with existing usage, but if our intent is to enforce a size of 20 this line will allow clients to deviate from that.

Reply:
I will add an issue, but just to be clear this was already possible by setting text attributes.